### DISCUSSION TITLE
Double/curly underlining?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#17038] [General]
I've seen references to Double/curly underlining (and others).  Are the control sequences documented anywhere?

**Comment by lhecker** [comment]
I believe kitty introduced these sequences; they're documented here: https://sw.kovidgoyal.net/kitty/underlines/
This has a few examples for color as well: https://github.com/microsoft/vscode/issues/158827

**Comment by vefatica** [comment]
Thanks, Leonard.  They work in WT.  Aren't they mentioned any where in this forum or in any WT docs?  How is one to know about them?
